zerver/lib/actions.py

```diff
@@ -1759,15 +1759,14 @@ def do_change_password(user_profile, password, commit=True,
                                  modified_user=user_profile, event_type='user_change_password',
                                  event_time=event_time)
 
-def do_change_full_name(user_profile, full_name, log=True):
-    # type: (UserProfile, Text, bool) -> None
+def do_change_full_name(user_profile, full_name, acting_user):
```

Comment:
yeah, it's going to be a more complicated change then -- we should change `check_change_full_name` to take the acting user, and pass it through. The acting user should only be None when it is being done by management command.